src/create_disk.sh

```diff
@@ -62,6 +68,19 @@ udevtrig() {
 export PATH=$PATH:/sbin:/usr/sbin
 arch="$(uname -m)"
 
+if [ -n "$platforms_json" ]; then
+    platform_grub_cmds=$(jq -r ".${arch}.${platform}.grub_commands // [] | join(\"\\\\n\")" < "${platforms_json}")
+    platform_kargs=$(jq -r ".${arch}.${platform}.kernel_arguments // [] | join(\" \")" < "${platforms_json}")
+else
+    # Add legacy kargs and console settings
+    platform_grub_cmds='serial --speed=115200\nterminal_input serial console\nterminal_output serial console'
+    DEFAULT_TERMINAL=$(. $(dirname "$0")/cmdlib.sh; echo $DEFAULT_TERMINAL)
+    platform_kargs="console=tty0 console=${DEFAULT_TERMINAL},115200n8"
```

Comment:
Previously, on s390x and aarch64 we would unset `tty` so that `DEFAULT_TERMINAL` wouldn't be injected. Seems like we're losing that here. Is that on purpose?

Reply:
It isn't.  Good catch!